Habitica/res/values/strings.xml

```diff
@@ -346,6 +346,7 @@ To start, which parts of your life do you want to improve?</string>
     <string name="opt_out_class">Opt Out</string>
     <string name="opt_out_description">Can\'t be bothered with classes? Want to choose later? Opt out - you\'ll be a warrior with no special abilities. You can read about the class system later on the wiki and enable classes at any time.</string>
     <string name="class_confirmation" formatted="false">Are you sure you want to be a %s?</string>
+    <string name="class_changed">You are now a %s, and you have a new default battle gear !</string>
```

Comment:
Yes, that's fine! In fact it's better than having it all in one sentence.

Can you remove the spaces from before the exclamation marks and say "You have new..." rather than "You have a new..."? "Gear" when used like this is a plural noun.